pyproject.toml

```diff
@@ -1,58 +1,47 @@
-# Copyright (C) 2022 Obscurity Labs LLC. <admin@obscuritylabs.com> - All Rights Reserved
-# Unauthorized copying of this file, via any medium is strictly prohibited.
-# All rights reserved. No warranty, explicit or implicit, provided.
-# Proprietary and confidential.
-
 [tool.poetry]
-authors = ["Obscurity Labs"]
-description = "Project DarkMoon - Web API service"
 name = "darkmoon"
-version = "0.0.1"
+version = "0.0.2"
+authors = ["Obscurity Labs"]
+description = "Project DarkMoon - CLI tool"
+readme = "README.md"
```

Comment:
We need to specify our project license in here, which also requires us to identify conflicting licenses in our dependencies and resolve them.